doc/src/manual/functions.md

```diff
@@ -1,4 +1,4 @@
-# [Functions](@id man-functions)
+# [Functions](https://docs.julialang.org/en/stable/manual/functions/)
```

Comment:
![screen shot 2018-01-13 at 11 36 13 am](https://user-images.githubusercontent.com/330950/34907978-02efd848-f856-11e7-988c-ff10bb547c7d.png)

This seems to have been working correctly for me already (and we don't want to embed full urls in the docs).